episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+values. There are bars of similar height across many different values (rather 
+than just one big peak at the left hand side).
+
+Image histograms are a great way to quickly summarise and compare pixel values 
+of different images.
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Histograms
+
+Match each of these small test images to their corresponding histogram. You can 
+assume that all images are displayed with a standard gray colormap and default 
+contrast limits of the min/max possible pixel values:
+
+![](fig/exercise-hist-images.png
+){alt="Screenshot of 4 small grayscale test images" width='68%'}
+
+![](fig/exercise-histograms.png
+){alt="Screenshot of 4 histograms, corresponding to the test images"}
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+- a - 3
+- b - 4
+- c - 2
+- d - 1
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Changing display settings
+
+What happens to pixel values when we change the display settings? Try changing 
+the contrast limits or colormap in the layer controls. You should see that the 
+histogram stays the same, no matter what settings you change i.e. the display 
+settings don't affect the underlying pixel values.
+
+This is one of the reasons it's important to use software designed for 
+scientific analysis to work with your light microscopy images. Software like 
+Napari and ImageJ will try to ensure that your pixel values remain unchanged, 
+while other image editing software (designed for working with photographs) may 
+change the pixel values in unexpected ways. Also, even with scientific software, 
+some image processing steps (that we'll see in later episodes) will change the 
+pixel values. 
+
+Keep this in mind and make sure you always retain a copy of your original data, 
+in its original file format! We'll see in episode X that original image files 
+contain important metadata that should be kept for future reference.
+
+## Colormaps / LUTs
+
+Let's dig deeper into Napari's colormaps. We've already seen in episode X that 
+the colormap can be changed by choosing different options from the 'colormap' 
+menu in the layer controls. But how do these colormaps actually work?
+
+Colormaps (also known as lookup tables or LUTs) match each pixel value to a 
+colour to use for display. For example, see the diagram below showing the 
+'gray' colormap:
+
+![](fig/gray-colorbar.png
+){alt="Grey colormap shown as a colorbar with corresponding pixel values" 
+width='50%'}
+
+Each pixel value from 0-255 maps to a corresponding colour. See the diagram 
+below for examples of 4 different colormaps with an example image histogram:
+
+![](fig/colorbar-comparison.png
+){alt="Four histograms showing the gray, green, viridis and inferno colormap 
+and correspoinding colorbar"}
+
+Why would we want to use different colormaps?
+
+- to highlight specific features in an image
+- to help with overlaying multiple images, as we saw in episode X when we 
```

Comment:
```suggestion
- to help with overlaying multiple images, as we saw in episode FIXME X when we 
```

Adding a `FIXME` may be easier for later to find out about it. (VS Code - through [todo-tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) and other editor have plug-ins that help to list these)
In any case, @milanmlft is there something in sandpaper that can refer to different episodes and link them so that it numbers it when compiled?

Reply:
Hmm, don't think so, but you can use [markdown internal links](https://carpentries.github.io/sandpaper-docs/episodes.html#internal-links), so instead maybe you could do

```suggestion
- to help with overlaying multiple images, as we saw in the [FIXME X episode](./FIXME-X.md) when we 
```

Personally I prefer the episode title being written out anyway, instead of having to remember what episode `<arbitrary-number>` refers to 😅